stores/sql/sqlite/main.go

```diff
@@ -258,3 +461,62 @@ func (tx *MainDatabaseTx) RenameObjects(ctx context.Context, bucket, prefixOld,
 	}
 	return nil
 }
+
+func (tx *MainDatabaseTx) fetchUsedContracts(ctx context.Context, fcids []types.FileContractID) (map[types.FileContractID]ssql.UsedContract, error) {
```

Comment:
SQL and MySQL implementations are identical, I feel we should really try to avoid duplicating this type of stuff. Maybe we can add a `main.go` under the `sql` directory to house a `fetchUsedContracts` that we call in both sql and mysql main files? This will likely not be the only shared impl. either... aside from deduplicating code it has the benefit of very clearly highlighting the differences between sql and mysql because any method where we can't call into the common impl. has a special sql quirk that prevents us from doing it.